lib/components/Atom/Calendar/date-picker.stories.tsx

```diff
@@ -0,0 +1,583 @@
+            control: { type: null },
+        },
+        wrapperClassName: {
+            table: {
+                type: {
+                    summary: "string | undefined",
+                },
+            },
+            description: "Class that will be applied to a calendar wrapper",
+            control: { type: "text" },
+        },
+    },
+} satisfies Meta<typeof DatePicker>;
+
+export default meta;
+type Story = StoryObj<typeof meta>;
+
+const Template: React.FC<Template> = ({ ...args }: Template) => {
+    const [formattedDate, setFormattedDate] = useState("");
+
+    return (
+        <>
+            <Text as="div">Selected date: {formattedDate}</Text>
+            <div style={{ width: `${args.hasFixedWidth ? "312px" : "90vw"}` }}>
+                <DatePicker
+                    {...args}
+                    onFormattedDate={(value) => setFormattedDate(value)}
+                />
+            </div>
+        </>
+    );
+};
+export const DefaultDatePicker = Template.bind(this) as Story;
+DefaultDatePicker.args = { ...meta.args };
+
+export const DatePickerWithRangeSelection = Template.bind(this) as Story;
+DatePickerWithRangeSelection.args = {
+    ...meta.args,
+    selectRange: true,
+};
+
+export const DatePickerWithSelectedValuePassed = Template.bind(this) as Story;
+DatePickerWithSelectedValuePassed.args = {
+    ...meta.args,
+    value: new Date(),
+};
+
+export const DatePickerWithDoubleNavigation = Template.bind(this) as Story;
+DatePickerWithDoubleNavigation.args = {
+    ...meta.args,
+    next2Label: (
+        <LabelPairedChevronsRightSmFillIcon fill="var(--component-textIcon-normal-prominent)" />
+    ),
+    prev2Label: (
+        <LabelPairedChevronsLeftSmFillIcon fill="var(--component-textIcon-normal-prominent)" />
+    ),
+};
+
+export const DatePickerWithCustomConfigForFormattingSelectedDate =
+    Template.bind(this) as Story;
+DatePickerWithCustomConfigForFormattingSelectedDate.args = {
+    ...meta.args,
+    optionsConfig: {
+        day: "numeric",
+        month: "short",
+        year: "numeric",
+    },
+};
+
+export const DatePickerWithCustomMonthAndYearFormatting = Template.bind(
+    this,
+) as Story;
+DatePickerWithCustomMonthAndYearFormatting.args = {
+    ...meta.args,
+    formatMonthYear: (locale, date) =>
+        new Date(date).toLocaleString(locale || navigator.languages, {
+            month: "2-digit",
+            year: "2-digit",
+        }),
+};
+
+export const DatePickerWithoutNavigation = Template.bind(this) as Story;
+DatePickerWithoutNavigation.args = {
+    ...meta.args,
+    showNavigation: false,
+};
+
+export const DatePickerWithNeighboringMonth = Template.bind(this) as Story;
+DatePickerWithNeighboringMonth.args = {
+    ...meta.args,
+    showNeighboringMonth: true,
+};
+
+export const DatePickerWithCustomActiveStartDate = Template.bind(this) as Story;
+DatePickerWithCustomActiveStartDate.args = {
+    ...meta.args,
+    startDate: new Date(2017, 0, 1),
```

Comment:
what do you think about `new Date('01/01/2017')` ? :)

Reply:
Sure, why not, let me push the commit